tests/console/snapper_cleanup.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use utils;
+use testapi;
+
+my $snapper_create .= "
+    snapper create --cleanup number --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number -u important=yes --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number -u important=yes --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number -u important=yes --command 'dd if=/dev/urandom of=data bs=1M count=1024'
+    snapper create --cleanup number -u important=yes --command 'dd if=/dev/urandom of=data bs=1M count=1024'";
```

Comment:
Yes, it will work with whatever count=N but the point is to create large files and test if snapper can keep the 50% as free disk space available.